benchmarking/cluster3nodes/configuration/log-config-generator.yaml

```diff
@@ -0,0 +1,149 @@
+# global filter; messages must have at least this severity to pass:
```

Comment:
@CodiePP, I think it's no longer appropriate to call these `log-config`'s -- they're general configuration since the recent changes.

Reply:
yes, we might rename these in the future